pkg/config/config.go

```diff
@@ -1361,6 +1361,10 @@ func load(config Config, origin string, loadSecret bool) (*Warnings, error) {
 	}()
 
```

Comment:
If I understood, in serverless env calling `config.ReadInConfig()` is not needed?

in that case why no doing
```golang
if IsServerless() { return &warnings, nil }
if err := config.ReadInConfig(); err != nil {
...
```

Reply:
Yeah, that's a good point. Done 👍